repo_files/documentations/bar_chart.md

```diff
@@ -88,11 +88,12 @@ enum values {`start`, `end`, `center`, `spaceEvenly`, `spaceAround`, `spaceBetwe
  |tooltipBgColor|background color of the tooltip bubble|Colors.white|
  |tooltipRoundedRadius|background corner radius of the tooltip bubble|4|
  |tooltipPadding|padding of the tooltip|EdgeInsets.symmetric(horizontal: 16, vertical: 8)|
- |tooltipBottomMargin|bottom margin of the tooltip (to the top of most top spot)|16|
+ |tooltipMargin|bottom margin of the tooltip (to the top of most top spot)|16|
```

Comment:
It is not **bottom** margin anymore.
Please update the text.

Reply:
Done.